ubuntu_desktop_installer/lib/l10n/app_localizations.dart

```diff
@@ -0,0 +1,327 @@
+  ],
+  'STANDALONEWEEKDAYS': <dynamic>[
+    'søndag',
+    'mandag',
+    'tirsdag',
+    'onsdag',
+    'torsdag',
+    'fredag',
+    'lørdag',
+  ],
+  'SHORTWEEKDAYS': <dynamic>[
+    'søn.',
+    'man.',
+    'tir.',
+    'ons.',
+    'tor.',
+    'fre.',
+    'lør.',
+  ],
+  'STANDALONESHORTWEEKDAYS': <dynamic>[
+    'søn.',
+    'man.',
+    'tir.',
+    'ons.',
+    'tor.',
+    'fre.',
+    'lør.',
+  ],
+  'NARROWWEEKDAYS': <dynamic>[
+    'S',
+    'M',
+    'T',
+    'O',
+    'T',
+    'F',
+    'L',
+  ],
+  'STANDALONENARROWWEEKDAYS': <dynamic>[
+    'S',
+    'M',
+    'T',
+    'O',
+    'T',
+    'F',
+    'L',
+  ],
+  'SHORTQUARTERS': <dynamic>[
+    'K1',
+    'K2',
+    'K3',
+    'K4',
+  ],
+  'QUARTERS': <dynamic>[
+    '1. kvartal',
+    '2. kvartal',
+    '3. kvartal',
+    '4. kvartal',
+  ],
+  'AMPMS': <dynamic>[
+    'a.m.',
+    'p.m.',
+  ],
+  'DATEFORMATS': <dynamic>[
+    'EEEE d. MMMM y',
+    'd. MMMM y',
+    'd. MMM y',
+    'dd.MM.y',
+  ],
+  'TIMEFORMATS': <dynamic>[
+    'HH:mm:ss zzzz',
+    'HH:mm:ss z',
+    'HH:mm:ss',
+    'HH:mm',
+  ],
+  'AVAILABLEFORMATS': null,
+  'FIRSTDAYOFWEEK': 0,
+  'WEEKENDRANGE': <dynamic>[
+    5,
+    6,
+  ],
+  'FIRSTWEEKCUTOFFDAY': 3,
+  'DATETIMEFORMATS': <dynamic>[
+    '{1} {0}',
+    '{1} \'kl\'. {0}',
+    '{1}, {0}',
+    '{1}, {0}',
+  ],
+};
+
+class _UbuntuLocalizationsDelegate
+    extends LocalizationsDelegate<MaterialLocalizations> {
+  const _UbuntuLocalizationsDelegate();
+
+  /// Add Additional languages (not supported by [GlobalMaterialLocalizations]) here.
+  @override
+  bool isSupported(Locale locale) =>
+      <String>['oc'].contains(locale.languageCode);
+
+  @override
+  Future<MaterialLocalizations> load(Locale locale) async {
+    // since date-format for custom locales is not required
+    const defaultLocaleName = 'en';
+    final localeName = Intl.canonicalizedLocale(locale.toString());
+
+    initializeDateFormattingCustom(
+      locale: defaultLocaleName,
+      patterns: _localeDatePatterns,
+      symbols: DateSymbols.deserializeFromMap(_nnDateSymbols),
+    );
+
+    return SynchronousFuture<MaterialLocalizations>(
+      GlobalUbuntuLocalizations(
+        localeName: localeName,
+        decimalFormat: NumberFormat('#,##0.###', 'en_US'),
+        twoDigitZeroPaddedFormat: NumberFormat('00', 'en_US'),
+        fullYearFormat: DateFormat('y', defaultLocaleName),
+        compactDateFormat: DateFormat('yMd', defaultLocaleName),
+        shortDateFormat: DateFormat('yMMMd', defaultLocaleName),
+        mediumDateFormat: DateFormat('EEE, MMM d', defaultLocaleName),
+        longDateFormat: DateFormat('EEEE, MMMM d, y', defaultLocaleName),
+        yearMonthFormat: DateFormat('MMMM y', defaultLocaleName),
+        shortMonthDayFormat: DateFormat('MMM d'),
+      ),
+    );
+  }
+
+  @override
+  bool shouldReload(_UbuntuLocalizationsDelegate old) => false;
+}
+
+class GlobalUbuntuLocalizations extends MaterialLocalizationEn {
```

Comment:
This approach doesn't look right: some code appears to be copy/pasted from an implementation of the 'nn' locale (Norwegian Nynorsk), and extending `MaterialLocalizationEn` [isn't recommended](https://api.flutter.dev/flutter/flutter_localizations/GlobalMaterialLocalizations-class.html#overriding-translations).

For occitan support, maybe we can extend [`MaterialLocalizationCa`](https://api.flutter.dev/flutter/flutter_localizations/MaterialLocalizationCa-class.html)? @Mejans what do you think? Would that be close enough with regards to date/time and number formats?

Reply:
yes, linguistically one language, politically two :) So yes it would be fine.

Reply:
No problem, As I said it was a dummy implementation (a quick fix).